models/incidents.go

```diff
@@ -212,6 +213,8 @@ func (a ActionType) String() string {
 		return "ForceClosed"
 	case ActionPurge:
 		return "Purged"
+	case ActionNote:
+		return "Note"
```

Comment:
@captncraig Should I change these to "Added note" so the action notification looks betters? Reference:

```
func init() {
	subject := `{{$first := index .States 0}}{{$count := len .States}}
{{.User}} {{.ActionType}}
{{if gt $count 1}} {{$count}} Alerts. 
{{else}} Incident #{{$first.Id}} ({{$first.Subject}}) 
{{end}}`
	body := `{{$count := len .States}}{{.User}} {{.ActionType}} {{$count}} alert{{if gt $count 1}}s{{end}}: <br/>
<strong>Message:</strong> {{.Message}} <br/>
<strong>Incidents:</strong> <br/>
<ul>
	{{range .States}}
		<li>
			<a href="{{$.IncidentLink .Id}}">#{{.Id}}:</a> 
			{{.Subject}}
		</li>
	{{end}}
</ul>`
	actionNotificationSubjectTemplate = ttemplate.Must(ttemplate.New("").Parse(strings.Replace(subject, "\n", "", -1)))
	actionNotificationBodyTemplate = htemplate.Must(htemplate.New("").Parse(body))
}
```

in notify.go